config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/stakater/workshop-operator
```

Comment:
i have pass image name in makefile IMG ?= quay.io/stakater/workshop-operator:v$(VERSION)